cmd/neofs-cli/modules/container/nodes.go

```diff
@@ -0,0 +1,63 @@
+package container
+
+import (
+	"crypto/sha256"
+
+	internalclient "github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	netmap2 "github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/netmap"
+	containerAPI "github.com/nspcc-dev/neofs-sdk-go/container"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/nspcc-dev/neofs-sdk-go/netmap"
+	"github.com/spf13/cobra"
+)
+
+var short bool
+
+var containerNodesCmd = &cobra.Command{
+	Use:   "nodes",
+	Short: "Show nodes for container",
+	Long:  "Show list of nodes as a result of applying container policy on the network map.",
```

Comment:
Show nodes taiking part in a container at the current epoch?

Reply:
No, you right, removed from public

Reply:
No, I am just rephrasing. We fetch current netmap and print container nodes.

Reply:
Gotcha, description changed.